engine/src/org/pentaho/di/trans/steps/pentahoreporting/PentahoReportingOutput.java

```diff
@@ -221,33 +225,54 @@ private void processReport( Object[] r, String sourceFilename, String targetFile
         }
       }
 
-      switch ( outputProcessorType ) {
+      ModifiableConfiguration modifiableConfiguration = (ModifiableConfiguration) report.getConfiguration();
+      String property;
+      Runnable exportTask;
+      switch( outputProcessorType ) {
```

Comment:
@deinspanjer, thak you for reviewing.

ExportTask classes are a part of pentaho-reporting. I don't really think it is a good idea to modify PRD for Kettle purposes. Instead, we should utilize PRD's classes in the same manner. The only think I'm concerned I bit is that these values (properties) are not defined as constants, but it is how they are written in PRD.

So, the fix is all about unifying approaches. Previously, Kettle used utility methods, whereas PRD itself moved to exporting tasks.